libs/gi/db/src/Database/DataManagers/TeamDataManager.ts

```diff
@@ -14,6 +14,7 @@ export interface Team {
   >
 
   teamCharIds: string[]
+  lastEdit: number
```

Comment:
Does this cause a bad rerender dependency every time an edit is made?

Reply:
what do you mean by "bad"?